readthedocs/config/config.py

```diff
@@ -361,12 +361,9 @@ def validate_python(self):
         version = self.defaults.get('python_version', 2)
         python = {
             'use_system_site_packages': use_system_packages,
```

Comment:
I think we should use `system_packages` as name here

> There are more context in another comment.

Reply:
Shorter is  better p: